### ISSUE TITLE
"npm install <package-name>" better than "npm install <module-name>"

### ISSUE THREAD

**Issue by cuppar** [npm/how-to-npm#140]
If the folder can be published to https://registry.npmjs.com, it must be contain a package.json file, so it must be a "package".
If the package has neither index.js file nor the "main" field in package.json, then it is not a module.
So, I think the "npm install <package-name>" better than "npm install <module-name>".

**Comment by kamal126** [comment]
I agree with the original suggestion. Referring to installable items from the npm registry as "packages" is more accurate than calling them "modules".

Every npm installable item must have a `package.json`, which defines it as a package. However, not all packages expose a module (i.e., they might not have an `index.js` or a `"main"` field), so calling them "modules" can be misleading, especially for beginners.

I’d suggest updating relevant documentation or CLI messages to consistently use "package" unless referring to the module system specifically.
